### PR TITLE
Implement move functionality

### PR DESCRIPTION
**Pull request by ln427**
https://trello.com/c/aGGll5Ri/28-implement-move

### CODE REVIEW

File: newbank/server/Customer.java [hunk 1]
```diff
@@ -3,13 +3,13 @@
 import java.util.ArrayList;
 
 public class Customer {
-	
+
 	private ArrayList<Account> accounts;
```
Comment (bairdj) [inline newbank/server/Customer.java]: This needs to be publicly accessible. I think a getter method should be added

File: newbank/server/Customer.java [hunk 2]
```diff
@@ -19,6 +19,16 @@ public String accountsToString() {
 	}
 
 	public void addAccount(Account account) {
-		accounts.add(account);		
+		accounts.add(account);
+	}
+
+	public boolean move(double amount, Account origAccount, Account destAccount) {
+		if (origAccount.balance >= amount) {
+			origAccount.updateBalance(origAccount.balance - amount);
```
Comment (bairdj) [inline newbank/server/Customer.java]: This should use the balance() method

File: newbank/server/Account.java [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
 		this.accountName = accountName;
 		this.openingBalance = openingBalance;
 	}
-	
+
 	public String toString() {
 		return (accountName + ": " + openingBalance);
 	}
 
+	public double balance(){
```
Comment (bairdj) [inline newbank/server/Account.java]: Should we use getBalance(), setBalance() etc. for consistency and clarity?

File: newbank/server/Customer.java (hunk 2 above)
Comment (bairdj) [reply newbank/server/Customer.java]: Or getBalance() etc.

Comment (ln427) [reply newbank/server/Customer.java]: Agree!

File: newbank/server/Account.java (hunk 3 above)
Comment (ln427) [reply newbank/server/Account.java]: 💯

File: newbank/server/Customer.java (hunk 1 above)
Comment (ln427) [reply newbank/server/Customer.java]: Correct!

**Comment by ln427** [comment]
Hi @bairdj, nice review. I updated the PR based on your comments.

**Comment by bairdj** [comment]
There is just one other part which is adding the MOVE command to the input processing, so that the user can use the functionality

**Comment by JaneMansell** [comment]
Are you going to add a MOVE case to the switch statement in NewBank.java so that the customer can implement the command?

**Comment by JaneMansell** [comment]
Snap! James has asked the same question.

**Comment by ln427** [comment]
@bairdj @JaneMansell PR updated!

**Comment by JaneMansell** [comment]
When I try and run your code I get this error:
image

**Comment by ln427** [comment]
@JaneMansell sorry for that, I pushed a fix for that.

**Comment by ln427** [comment]
Screen Shot 2020-03-15 at 17 49 09

**Pull request merged**